mobile-app/App.js

```diff
@@ -63,7 +62,8 @@ const PassengerScreens = ({route, navigation}) => {
             initialParams={{
               "region": route.params.region,
               "position": route.params.position, 
-              "user": dummyPassenger.KeaG7oWA9tGo067gcByO, 
+              "user": dummyPassenger["63MwWi3KaQxECaQbAoEn"],
+              "userID":  "63MwWi3KaQxECaQbAoEn",
```

Comment:
Should we replace "63MwWi3KaQxECaQbAoEn" with Object.keys(dummyPassenger)[0] so we can just pass or replace dummyPassenger